components/camel-mllp/src/test/java/org/apache/camel/component/mllp/MllpEndpointTest.java

```diff
@@ -16,33 +16,38 @@
  */
 package org.apache.camel.component.mllp;
 
-import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
 
 /**
  * Tests for the MllpEndpoint class.
+ *
+ * @author vishal423
```

Comment:
Please remove the @author doc, we don't use it. Thanks

Reply:
sure